worlds/AutoWorld.py

```diff
@@ -3,16 +3,18 @@
 import hashlib
 import logging
 import pathlib
-import random
 import re
 import sys
 import time
 from dataclasses import make_dataclass
 from typing import (Any, Callable, ClassVar, Dict, FrozenSet, List, Mapping,
                     Optional, Set, TextIO, Tuple, TYPE_CHECKING, Type, Union)
 
-from Options import PerGameCommonOptions
-from BaseClasses import CollectionState
+from Options import ExcludeLocations, ItemLinks, LocalItems, NonLocalItems, PerGameCommonOptions, \
+    PriorityLocations, \
+    StartHints, \
+    StartInventory, StartInventoryPool, StartLocationHints
+from BaseClasses import CollectionState, OptionGroup
 
 if TYPE_CHECKING:
     import random
```

Comment:
Something's up with random in the tests, and I'm gonna assume it's here, but I'm not sure